app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index;end
+
+  def show
+    @user = User.find(params[:id])
+    render "user/show"
+  end
+
+  def edit
+    @user = current_user
+    render "user/edit"
```

Comment:
Do we need it?

Reply:
Yes, this is to update only the password. It was one of the requirements on the project.